DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/_DelegateCourseInfoCard.cshtml

```diff
@@ -0,0 +1,132 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@model DelegateCourseInfoViewModel
+
+<div class="nhsuk-expander-group" id="@Model.CustomisationId-card">
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.CustomisationId-name" name="name">
+        @Model.ApplicationName - @Model.CustomisationName
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <dl class="nhsuk-summary-list details-list-with-button">
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Enrolled
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.Enrolled" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Supervisor
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.Supervisor" />
+          <dd class="nhsuk-summary-list__actions">
+            <a href="#">Edit<span class="nhsuk-u-visually-hidden"> supervisor</span></a>
+          </dd>

```

Comment:
From the screenshots, it looks like the presence of the actions messes up the line separators between the fields. The wireframes have uniform lines that go all the way across on every field.